### PR TITLE
refactor(ci): get rid of bazel build

### PR DESCRIPTION
**Pull request by universal-itengineer**
## Description

Switch to building kubevirt components on go and werf instead of building in bazel and werf

## Why do we need it, and what problem does it solve?

We need to simplify the build, make it more predictable and possible in closed environments

## What is the expected result?

All pods with prefix `virt-` in `d8-virtualization` will be restarted
All VM `'firmware'` will be updated by vm migration 

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: images/libguestfs/werf.inc.yaml [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
   - .version
-  - entrypoint.sh
-  to: /
   before: setup
 shell:
+  beforeInstall:
+  - apt-get update && apt-get install --yes curl
+  - mkdir -p /usr/local/lib/guestfs/
+  - curl -L {{ $libguestfsApplianceTar }} | tar -xJ -C /usr/local/lib/guestfs/
+  - apt-get remove curl --yes
```
Comment (diafour) [inline images/libguestfs/werf.inc.yaml:25]: Append `apt clean` and `rm -rf` commands to reduce stage image size.

Comment (universal-itengineer) [reply images/libguestfs/werf.inc.yaml:25]: Fixed

File: images/virt-launcher/werf.inc.yaml [hunk 2]
```diff
@@ -114,6 +117,7 @@ shell:
     seabios \
     libffi8 \
     swtpm-tools \
+    glibc \
```
Comment (diafour) [inline images/virt-launcher/werf.inc.yaml:120]: Seems redundant. glibc should be installed in the base image.

Comment (universal-itengineer) [reply images/virt-launcher/werf.inc.yaml]: You're right.
Fixed.

**Pull request merged**